certified-connectors/gofileroom/settings.json

```diff
@@ -0,0 +1,9 @@
+{

```

Comment:
@AlfredoDinunzioTR The settings.json file is not required, Please remove it from the PR.